doc/manual/documentation.rst

```diff
@@ -494,15 +494,340 @@ Examining it's definition should serve as an example of how to use ``@__doc__``
+   > technical computing environments.
+
+Note that a single space must appear after the ``>`` character on each line. Quoted blocks may themselves contain other toplevel or inline elements.
+
+Images
+^^^^^^
+
+The syntax for images is similar to the link syntax mentioned above. Prepending a ``!`` character to a link will display an image from the specified URL rather than a link to it.
+
+.. code-block:: text
+
+   ![alternative text](link/to/image.png)
+
+Lists
+^^^^^
+
+Unordered lists can be written by prepending each item in a list with either ``*``, ``+``, or ``-``.
+
+.. code-block:: text
+
+   A list of items:
+
+     * item one
+     * item two
+     * item three
+
+Note the two spaces before each ``*`` and the single space after each one.
+
+Lists can contain other nested toplevel elements such as lists, code blocks, or quoteblocks. A blank line should be left between each list item when including any toplevel elements within a list.
+
+.. code-block:: text
+
+   Another list:
+
+     * item one
+
+     * item two
+
+       ```
+       f(x) = x
+       ```
+
+     * And a sublist:
+
+         + sub-item one
+         + sub-item two
+
+.. note::
+
+    The contents of each item in the list must line up with the first line of the item. In the above example the fenced code block must be indented by four spaces to align with the ``i`` in ``item two``.
+
+Ordered lists are written by replacing the "bullet" character, either ``*``, ``+``, or ``-``, with a positive integer followed by either ``.`` or ``)``.
+
+.. code-block:: text
+
+   Two ordered lists:
+
+    1. item one
+    2. item two
+    3. item three
+
+
+    5) item five
+    6) item six
+    7) item seven
+
+An ordered list may start from a number other than one, as in the second list of the above example, where it is numbered from five. As with unordered lists, ordered lists can contain nested toplevel elements.
+
+Display equations
+^^^^^^^^^^^^^^^^^
+
+Large :math:`\LaTeX` equations that do not fit inline within a paragraph may be written as display equations using a fenced code block with the "language" ``math`` as in the example below.
+
+.. code-block:: text
+
+   ```math
+   f(a) = \frac{1}{2\pi}\int_{0}^{2\pi} (\alpha+R\cos(\theta))d\theta
+   ```
+
+Footnotes
+^^^^^^^^^
+
+This syntax is paired with the inline syntax for `Footnote references`_. Make sure to read that section as well.
+
+Footnote text is defined using the following syntax, which is similar to footnote reference syntax, aside from the ``:`` character that is appended to the footnote label.
+
+.. code-block:: text
+
+   [^1]: Numbered footnote text.
+
+   [^note]:
+
+       Named footnote text containing several toplevel elements.
+
+         * item one
+         * item two
+         * item three
+
+       ```julia
+       function func(x)
+           # ...
+       end
+       ```
+
+.. note::
+
+    No checks are done during parsing to make sure that all footnote references have matching footnotes.
```

Comment:
Yes, there's a "linting" stage in Documenter that this could slot into quite nicely I think.